projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.ts

```diff
@@ -49,6 +50,9 @@ export class TimelineCardListComponent implements AfterContentInit {
   @Input()
   public selectedIndex?: number;
 
+  @Input()
+  public hideTimeView?: boolean = false;
```

Comment:
could we flip this to something like showTime? I think it's easier to mentally parse boolean properties that are in the affirmative.

Reply:
Sure

Reply:
Addressed in the fixing-timeline-card-view PR